zdatasets/utils/aws.py

```diff
@@ -63,6 +60,46 @@ def get_aws_client(role_arn: str, service: str):
     return session.client(service)
 
 
+def send_sqs_message(
+    queue_url: str,
+    message_body: str,
+    *,  # keyword-only parameters
```

Comment:
is keyword-only parameters necessary given the following parameters are already keyword only?
Maybe I don't understand it's use.

Reply:
If without *, it is confusing what does `val3` means in this call, send_sqs_message(val1, val2, val3). '*' is telling that all parameters after it must be keyword-only.

Reply:
Must they not be named anyways because role_arn and profile_name are named?
Or are you introducing `*` as positional parameter to force any future parameters after it to be keyword only? :)